### PR TITLE
bugfix for BorrowCommand & Update userguide

### PR DESCRIPTION
**Pull request by MuhammadHoze**
1. Added Exceptions to BorrowCommand
2. Updated User Guide

### CODE REVIEW

File: src/main/java/seedu/duke/commands/CommandCreator.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package seedu.duke.commands;
 
+import jdk.jshell.execution.JdiExecutionControl;
```
Comment (iamchenjiajun) [inline src/main/java/seedu/duke/commands/CommandCreator.java]: what's this

File: src/main/java/seedu/duke/commands/CommandCreator.java [hunk 2]
```diff
@@ -235,11 +239,12 @@ public static Command createReturnCommand(String commandString) throws DukeExcep
         } catch (NumberFormatException e) {
             throw new DukeException(Messages.EXCEPTION_INVALID_INDEX);
         } catch (IndexOutOfBoundsException e) {
-            throw new DukeException(Messages.WARNING_NO_TASK);
+            throw new DukeException(Messages.WARNING_NO_BOOK);
         }
     }
 
     // @@author GuoAi
+
```
Comment (iamchenjiajun) [inline src/main/java/seedu/duke/commands/CommandCreator.java]: maybe this whitespace isn't needed

**Review by iamchenjiajun — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/commands/CommandCreator.java (hunk 1 above)
Comment (MuhammadHoze) [reply src/main/java/seedu/duke/commands/CommandCreator.java]: what's that?

Comment (MuhammadHoze) [reply src/main/java/seedu/duke/commands/CommandCreator.java]: removed

File: src/main/java/seedu/duke/commands/CommandCreator.java (hunk 2 above)
Comment (MuhammadHoze) [reply src/main/java/seedu/duke/commands/CommandCreator.java]: removed

**Review by iamchenjiajun — APPROVED** [review]

**Pull request merged**